x-pack/plugin/security/src/main/java/module-info.java

```diff
@@ -17,6 +17,7 @@
     requires org.elasticsearch.transport.netty4;
     requires org.elasticsearch.xcontent;
     requires org.elasticsearch.xcore;
+    requires io.netty.codec.http;
```

Comment:
super nit: can you move this down by the other io.netty requires ?

Reply:
pushed [0d03f6e](https://github.com/elastic/elasticsearch/pull/95112/commits/0d03f6e324ad98f82c2a56d13320d22c2300acfe).